docs/Contributing/architecture/orchestration/query-packs.md

```diff
@@ -0,0 +1,78 @@
+# Query Packs Architecture
```

Comment:
To my knowledge, query packs aren't pushed anymore for Fleet. Given that, should we be writing more documentation about them?

Reply:
They aren't pushed but are still supported so I think worth documenting their architecture.